packages/docs/src/styles/ecosystem-diagram.css

```
@@ -0,0 +1,543 @@
+}
+
+.eco-node-text {
+  display: flex;
+  min-width: 0;
+  flex-direction: column;
+  gap: 0.12rem;
+}
+
+.eco-node-name {
+  overflow: hidden;
+  font-family: 'SF Mono', 'Monaco', 'Consolas', 'Liberation Mono', monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Monaco" (font-family-name-quotes)

(font-family-name-quotes)